social_distribution/author_manager/views.py

```diff
@@ -289,15 +290,16 @@ def get(self, request, id):
             - If successful:
                 Status 200 and the author's basic information.
         """
-        profile = Author.objects.get(id=id)
-        # profile = get_object_or_404(Author, id=id)
+        # profile = Author.objects.get(id=id)
+        profile = get_object_or_404(Author, id=id)
         serializer = ProfileSerializer(profile, remove_fields=['user'] )
         return Response({'profile': profile})
```

Comment:
you should return serialize otherwise this API cannot be reached.

Reply:
That was because I combined api view and ui. 
I separate them, API can be reached now

Reply:
Not sure you meant you got it fixed locally or here. But if this is an API endpoint, it needs to return serializer instead of queryset in order to be rendered into JSON and understood by other nodes

Reply:
I've already fixed this in the most recent commit, pls take a look